omero/sysadmins/version-requirements.txt

```diff
@@ -1750,12 +1743,26 @@ Distribution support
       - N/A
       - N/A
     * - 1.6
-      - N/A
+      - EPEL
       - 21, (22)
       - 14.10, (15.04)
       - (8.0), 7.0 bpo
       - Yes
       - Yes
+    * - 1.8
+      - N/A
+      - N/A
+      - N/A
+      - N/A
+      - Yes
+      - Yes
+    * - 1.9
+      - N/A
+      - N/A
+      - 14.10
```

Comment:
note that we have now installed 1.8 on all OS in omero-install
For Ubuntu14.04, the "default" version installed is 1.4
if we wish to support < 1.8 we will have to introduce a new parameter